diff --git a/node_modules/vscode-cdp-proxy/dist/connection.js b/node_modules/vscode-cdp-proxy/dist/connection.js
index f46fe58..e934b5e 100644
--- a/node_modules/vscode-cdp-proxy/dist/connection.js
+++ b/node_modules/vscode-cdp-proxy/dist/connection.js
@@ -126,7 +126,7 @@ class Connection {
         if (this.pauseQueue) {
             this.pauseQueue.push(message);
         }
-        if (message.id === undefined) {
+        if (message.id === undefined || message.method !== undefined) {
             // for some reason, TS doesn't narrow this even though IProtocolCommand
             // is the only type of the tuple where id can be undefined.
             const asCommand = message;
diff --git a/node_modules/vscode-cdp-proxy/dist/index.js b/node_modules/vscode-cdp-proxy/dist/index.js
index ca7886c..be99d9e 100644
--- a/node_modules/vscode-cdp-proxy/dist/index.js
+++ b/node_modules/vscode-cdp-proxy/dist/index.js
@@ -16,7 +16,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 __exportStar(require("./server"), exports);
 __exportStar(require("./disposable"), exports);
 __exportStar(require("./connection"), exports);
-__exportStar(require("./cdp"), exports);
+// __exportStar(require("./cdp"), exports);
 __exportStar(require("./cdp-error"), exports);
 __exportStar(require("./transports/transports"), exports);
 __exportStar(require("./transports/websocket"), exports);
